com.pixelomer.ShijimaQt.yaml

```diff
@@ -0,0 +1,54 @@
+app-id: com.pixelomer.ShijimaQt
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: shijima-qt
+finish-args:
+  - --device=dri
+  - --socket=x11
+  - --share=ipc
+  - --env=SHIJIMA_FLATPAK=1
+  - --socket=pulseaudio
+  - --env=QT_QPA_PLATFORM=xcb
+  # To be able to listen to and communicate with port 32456 for HTTP API and CLI
+  - --share=network
+  # Required for activating/deactivating shell extensions
+  - --talk-name=org.gnome.Shell
+  - --talk-name=org.kde.KWin
+  # Required for running gnome-extensions install
+  - --talk-name=org.freedesktop.Flatpak
+
+modules:
+  - name: shijima-qt
+    buildsystem: simple
+    build-commands:
+      # Workaround for lack of yarn in build environment
+      # Will not be necessary after v0.0.4
+      - mkdir -p Platform/Linux/.work/node_modules/.bin
+      - echo -e '#!/usr/bin/env bash\ncat' > Platform/Linux/.work/node_modules/.bin/minify
+      - chmod +x Platform/Linux/.work/node_modules/.bin/minify
+
+      # Build
+      - CONFIG=release make -j`nproc`
+
+      # Install
+      - install -Dm755 publish/Linux/release/shijima-qt /app/bin/shijima-qt
+      - install -Dm755 publish/Linux/release/libunarr.so.1 /app/lib/libunarr.so.1
+      - install -Dm644 com.pixelomer.ShijimaQt.desktop /app/share/applications/com.pixelomer.ShijimaQt.desktop
+      - install -Dm644 flathub-com.pixelomer.ShijimaQt.metainfo.xml /app/share/metainfo/com.pixelomer.ShijimaQt.metainfo.xml
+      - install -Dm644 com.pixelomer.ShijimaQt.png /app/share/icons/hicolor/512x512/apps/com.pixelomer.ShijimaQt.png
+    sources:
+      # Source code
+      - type: git
+        url: https://github.com/pixelomer/Shijima-Qt.git
+        tag: v0.0.4
+      
+      # Metadata specifically for Flathub
```

Comment:
it's not specifically for flathub. Also is should be upstream.

Reply:
Should be resolved in [3e330d5](https://github.com/pixelomer/Shijima-Qt-flathub/commit/3e330d5a069d912f076bf8da6067c5f426b43aad).